README.md

```diff
@@ -30,13 +29,13 @@ Before building kaazing-client-javascript, please install and configure the foll
 
 You can also download and run this project from [kaazing.org/download](http://kaazing.org/download/)
 
-# Learning How to Develop Client Applications
+# Learning How to Use an RFC 6455 endpoint
 
-To learn how to develop client applications using the Gateway, see the documentation on [kaazing.org](http://kaazing.org).  To contribute to the documentation source, see the doc directory under each protocol's folder.
+You can use an RFC-6455 endpoint, such KAAZING Gateway. To learn how to administer the Gateway, its configuration files, and security, see the documentation on [developer.kaazing.com](http://developer.kaazing.com/documentation/5.0/index.html).
```

Comment:
This section can be used as the template across all client repos (to replace what you have currently, which is specific to the Gateway).